scrapers_next/or/committee.py

```diff
@@ -0,0 +1,61 @@
+from spatula import XPath, URL, HtmlListPage, HtmlPage, CSS
+from openstates.models import ScrapeCommittee
+import re
+
+class CommitteeDetail(HtmlListPage):
+    source = URL("https://olis.oregonlegislature.gov/liz/2023R1/Committees/JCT/CommitteeMembers")
+    selector = CSS('tbody tr')
+
+    def process_item(self, item):
+        com = self.input
+        position, name = item.getchildren()
+        name = re.sub(r"Senator |Representative ", "", name.text_content().strip())
+        position = position.text_content().strip()
+        com.add_member(name, position)
+
+        return com
+
+
+class CommitteeList(HtmlListPage):
+    source = "https://olis.oregonlegislature.gov/liz/Committees/list/"
+
+    def process_item(self, item):
+        link = item.get("href").split('/')
+        link = 'https://olis.oregonlegislature.gov/liz/' + link[4] +\
+                "/Committees/" + link[6] + "/CommitteeMembers"
+
+        try:
+            parent = self.parent
+        except:
```

Comment:
It's good practice to specify the type of exception you're allowing for in your `except` clause.

In this case, when you don't have `parent` specified, you get an `AttributeError`. So you can make this line:

`except AttributeError:`